contracts/adapters/Ragequit.sol

```diff
@@ -105,9 +111,7 @@ contract RagequitContract is IRagequit, DaoConstants, MemberGuard {
         // it considers the locked loot to be able to calculate the fair amount to ragequit,
         // but locked loot can not be burned.
         uint256 initialTotalSharesAndLoot =
-            bank.balanceOf(TOTAL, SHARES) +
-                bank.balanceOf(TOTAL, LOOT) +
-                bank.balanceOf(TOTAL, LOCKED_LOOT);
```

Comment:
Locked loot was remove to make it consistent with Guild Kick and Rage Quit, since we are not really using the Locked Loot.

Reply:
you need to use LOCKED_LOOT when you calculate the total amount, just not when you burn. Is LOCKED_LOOT not used to calculate the total amount of shares + loot in ragequit ?

Reply:
No. There we only use the SHARES and LOOT.

Reply:
Fixed. Just added the Locked Look to the calc, for RageQuit and GuildKick.